0000-interledger-eli5/0000-interledger-eli5.md

```diff
@@ -0,0 +1,124 @@
+
+_Note: In a real bank, it's possible for an account to hold multiple types of assets. For example, one bank account could hold amounts in US dollars, Euro's, and Japanese Yen. To keep things simple in the Interledger Protocol, each "ledger" is limited to recording how much different parties have **of exactly the same kind of asset**. If another asset is used by a sender or receiver, another ledger is kept for that asset. From ILP's point of view, a particular party may have several ledgers associated with them._
+
+### ILP Security: Trusted & Trustless
+
+When you have an asset stored somewhere, like money in a bank, what do you really have?  The thing you really have is _trust_ that when you go to that place and ask for your asset back, that it will be given to you.  In the case of a bank, you show them your bankcard and ask for your money.  Usually that process is automated: you put your card into a bank ATM, tell it you want your money, and out it comes.
+
+If you didn't trust a particular bank to give you your money back when you ask for it, you probably wouldn't give it to them in the first place.
+
+This is important because this idea of trust is at the core of how the pieces of ILP fit together, and - interestingly - how this arrangement enables _trustless payments_.  Let's see how it works:
+
+**Payment Setup: Being Prepared**
+
+As much as possible about an ILP payment is prepared in advance to be sure each party is happy with what's going to happen, and that the funds will remain safe while in transition.  While it's possible for a sender to make a payment to a receiver on the same ledger, examining that scenario won't involve much of what makes ILP interesting.  So let's take the example that the sender and receiver are on different ledgers.
+
+Setup goes like this:
+
+1) A sender asks a connector: "Do you know this receiver?"
+
+2) If the connector knows the receiver, great.  If not, the connector talks to its "peers" (its "connector friends") to see if any know the receiver, or know another connector that might know the receiver... and so on.  Through this process, a chain of connectors linking the sender to the receiver is discovered.  This chain may involve just one connector, or it may involve many.
+
+3) Connectors can each charge a little bit for their part in handling the payment.  After the chain is built, the connector closest the sender explains how much the receiver will get if the payment goes down that chain.  If it's much less than what the sender wants to send, the sender will probably choose a different connector.  So connectors try make the cost to the sender as little as possible, while still earning something.
+
+4) The sender decides whether or not to use that connector, based on how much it will cost.  If it's a "no", the sender can start over asking a different connector, or asking the same one again (who may figure out a better way this time).  When the sender is happy with the cost, the payment can go ahead.
```

Comment:
All of these steps technically describe the quoting phase, not the "Preparation" phase. Normally when we talk about payments being prepared, that means the money is being put on hold all the way down the path